editor/editor_node.cpp

```diff
@@ -3242,9 +3242,23 @@ Ref<Texture> EditorNode::get_class_icon(const String &p_class, const String &p_f
 		RES icon;
 		if (FileAccess::exists(icon_path)) {
 			icon = ResourceLoader::load(icon_path);
-		}
-		if (!icon.is_valid()) {
-			icon = gui_base->get_icon(ScriptServer::get_global_class_base(p_class), "EditorIcons");
+			if (icon.is_valid()) {
```

Comment:
This changes the previous logic, I guess you wanted `!icon.is_valid()`?

Reply:
Ping.

Reply:
I've amended the implementation. Recompiling on my end. Will upload again when next available.